libraries/Microsoft.BotKit.Adapters.Slack/SlackBotWorker.cs

```diff
@@ -36,10 +32,14 @@ public SlackBotWorker(Botkit botkit, BotWorkerConfiguration config)
             // TODO make overload to allow a teamid (string) to be passed in
         }
 
+        // public SlackClient api;
+        public SlackTaskClient ApiClient { get; set; }
+
         /// <summary>
         /// Switch a bot's context to a 1:1 private message channel with a specific user.
         /// After calling this method, messages sent with `bot.say` and any dialogs started with `bot.beginDialog` will occur in this new context.
         /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
```

Comment:
There is extra space and the first letter starting with lower-case. Maybe you forgot to mention the return type. Can you fix it? The same applies to all the <returns> comments